src/App.jsx

```diff
@@ -1,104 +1,32 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function getUser(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getComments(PostId) {
```

Comment:
Here it is better to rename the change according to camelCase usage.
```suggestion
function getComments(postId) {
```